images/sponsors/systemetric-hills-road.svg

```diff
@@ -0,0 +1,375 @@
+       sodipodi:nodetypes="cccscccsscccscscscsccscccccssssccscsscscscccsscccsccccc" />
+    <ellipse
+       id="path410"
+       style="fill:#ffaaaa;stroke:#000000;stroke-width:0.10176"
+       cx="68.824081"
+       cy="33.629044"
+       rx="0.019593023"
+       ry="0.018783854" />
+    <path
+       style="fill:#ffaaaa;fill-rule:evenodd;stroke-width:0.10176"
+       id="path414"
+       sodipodi:type="arc"
+       sodipodi:cx="56.037262"
+       sodipodi:cy="24.606306"
+       sodipodi:rx="1.7472672"
+       sodipodi:ry="1.4847541"
+       sodipodi:start="3.9634189"
+       sodipodi:end="3.8336219"
+       sodipodi:arc-type="arc"
+       d="m 54.847574,23.518886 a 1.7472672,1.4847541 0 0 1 2.430099,0.04173 1.7472672,1.4847541 0 0 1 0.02976,2.065262 1.7472672,1.4847541 0 0 1 -2.428167,0.09228 1.7472672,1.4847541 0 0 1 -0.18732,-2.059274"
+       sodipodi:open="true" />
+    <path
+       style="fill:#ffaaaa;fill-rule:evenodd;stroke-width:0.10176"
+       id="path416"
+       sodipodi:type="arc"
+       sodipodi:cx="63.790607"
+       sodipodi:cy="24.141314"
+       sodipodi:rx="1.5524604"
+       sodipodi:ry="1.3998399"
+       sodipodi:start="3.9634158"
+       sodipodi:end="3.8336157"
+       sodipodi:open="true"
+       sodipodi:arc-type="arc"
+       d="m 62.733557,23.116087 a 1.5524604,1.3998399 0 0 1 2.159161,0.03933 1.5524604,1.3998399 0 0 1 0.02645,1.947147 1.5524604,1.3998399 0 0 1 -2.157444,0.08702 1.5524604,1.3998399 0 0 1 -0.166448,-1.941501" />
+    <text
+       xml:space="preserve"
+       style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:48.9334px;line-height:1.25;font-family:'Niagara Engraved';-inkscape-font-specification:'Niagara Engraved';fill:#ffaaaa;fill-opacity:1;stroke:none;stroke-width:1.22333"
```

Comment:
It seems this SVG expects a certain font to be available. This should really be converted to an outline instead.

Reply:
Are you able to do that or should I be asking them for it?

Reply:
I'm able to do it. Turns out this is a font bundled with Windows.

Reply:
In fact we actually need to do this, looks like this if you don't have the font.
<img width="631" alt="image" src="https://user-images.githubusercontent.com/6197148/144724302-9936e1e5-823c-4776-b3c8-ee2f387365e7.png">